docker/Dockerfile

```diff
@@ -0,0 +1,235 @@
+#
+# Docker file that builds RelEx and visual question answering pipeline.
+#
+# To start:
+#    docker run -it -p 8888:8888 relex-vqa
+#
+# To demo:
+#    open in browser localhost:8888
+#    use password: password 
+#
+# That will open jupyter notebook with vqa demo
+#
+FROM ubuntu:18.04
+# use bash
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+# Avoid triggering apt-get dialogs (which may lead to errors). See:
+# https://stackoverflow.com/questions/25019183/docker-java7-install-fail
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV JAVA_HOME=/usr/lib/jvm/java-1.11.0-openjdk-amd64
+
+RUN apt-get update ; apt-get -y upgrade ; apt-get -y autoclean
+
+# Java
+RUN apt-get -y install maven screen telnet netcat-openbsd byobu \
+                       wget vim git unzip sudo apt-utils
+
+# incorrect packaging in ubuntu-xenial
+# dpkg: error processing archive /var/cache/apt/archives/openjdk-9-jdk_9~b114-0ubuntu1_amd64.deb (--unpack):
+# trying to overwrite '/usr/lib/jvm/java-9-openjdk-amd64/include/linux/jawt_md.h', which is also in package openjdk-9-jdk-headless:amd64 9~b114-0ubuntu1
+RUN apt-get -o Dpkg::Options::="--force-overwrite" -y install openjdk-11-jdk
+
+# GCC and basic build tools
+RUN apt-get -y install gcc g++ make swig ant
+
+# Wordnet
+RUN apt-get -y install wordnet wordnet-dev wordnet-sense-index
+
+# There are UTF8 chars in the Java sources, and the RelEx build will
+# break if build in a C environment.
+RUN apt-get -y install locales && locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+RUN mkdir /usr/local/share/java
+
+WORKDIR /home/Downloads/
+
+# JWNL - Never changes, so do this first.
+RUN wget https://downloads.sourceforge.net/project/jwordnet/jwnl/JWNL%201.4/jwnl14-rc2.zip \
```

Comment:
All java dependencies will be installed by maven automatically you don't need to install them manually.